nest-tax-backend/src/noris/subservices/noris-connection.subservice.ts

```diff
@@ -25,6 +25,24 @@ export class ConnectionSubservice {
     }
   }
 
+  async createConnection(): Promise<ConnectionPool>{
```

Comment:
`createConnection` and `createOptimizedConnection` should be only one function (the other can be an alias). `createConnection` should take two parameters for these values, with defaults set to the current `createConnection` values (120 and 120). Then, `createOptimizedConnection` should simply call `createConnection` with different parameters (60 and 180) - the client using `createOptimizedConnection` can also specify the 60 and 180 itself if applicable (in such case `createOptimizedConnection` could be removed altogether)